src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -127,47 +136,65 @@ private bool HasCallerMemberNameAttribute
                                                   && !HasCallerFilePathAttribute
                                                   && HasCallerMemberNameAttribute;
 
-        private ConstantValue DefaultSyntaxValue
+        private ConstantValue DefaultSyntaxValue => LazyDefaultSyntaxValue(out var unused, needsBoundExpression: false);
+
+        private ConstantValue LazyDefaultSyntaxValue(out BoundExpression boundExpressionOpt, bool needsBoundExpression)
```

Comment:
Could we discuss this more? The "relevant portion" of the method is nearly the entire method (technically the last two lines aren't needed), so I'm not sure what I would be extracting.